tests/src/dir.targets

```diff
@@ -120,7 +119,8 @@
 
   <Target Name="CopyNativeProjectBinaries">
      <ItemGroup>
-        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" />
+        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" Condition="'$(BuildOS)' == 'Windows_NT'"/>
+        <NativeProjectBinaries Include="$(ProjectDir)\..\bin\Product\$(OSPlatformConfig)\bin\*.*" Condition="'$(BuildOS)' != 'Windows_NT'"/>
```

Comment:
Do you mean I should leave only windows-line above and copy binaries somewhere else, in some shell script for example?